collections/app/controllers/CollectionsController.scala

```diff
@@ -126,11 +137,35 @@ object CollectionsController extends Controller with ArgoHelpers {
     } getOrElse Future.successful(invalidJson(req.body))
   }
 
+  type MaybeTree = Option[Node[Collection]]
+  def hasChildren(path: List[String]): Future[Boolean] =
+    allCollections.map { tree =>
```

Comment:
Does it make more sense to retrieve the collection by using dynamo.get? 

it seems like this is traversing all the collections to find the selected one, but I think they're all stored individually in dynamo, so it might be faster. Could also be nice to have a collections/:id endpoint that would use the same method, but maybe that doesnt matter.